docs/docs/action-server/actions.mdx

```diff
@@ -19,6 +19,18 @@ using a default location if the user doesn't specify one.
 The action will set the `temperature` to the maximum temperature of the weather forecast.
 The message returned will differ according to the channel they are using. 
 
+
+:::info Compressed body in HTTP requests
```

Comment:
The suggestions I left [here](https://github.com/RasaHQ/rasa/pull/11998#pullrequestreview-1297935262) apply to this PR too.

Reply:
[f0dcf1a](https://github.com/RasaHQ/rasa/pull/11997/commits/f0dcf1ab054cbedaf334293fad01c4c029a610e4) ✅